redstone/client.py

```diff
@@ -418,6 +418,14 @@ def create(self, name, payload=None, raw_payload=None, root=False):
         elif payload is not None:
             data['resources'][0]['payload'] = base64.b64encode(payload).decode('utf-8')
 
+        # TKEK related fields
+        if encryption is not None:
+            data['resources'][0]['encryptionAlgorithm'] = 'RSAES_OAEP_SHA_256'
+        if signedNonce is not None:
+            data['resources'][0]['signedNonce'] = signedNonce
+        if iv is not None:
+            data['resources'][0]['iv'] = iv
```

Comment:
I don't understand what these fields are... the docs indicate that there should be an `import_key` field...

https://cloud.ibm.com/docs/services/key-protect?topic=key-protect-import-root-keys#import-root-key

Also, if you need both you should have them pass a tuple of the (nonce, iv) pair or check both and raise a ValueError("'signed_nonce' and 'iv' must both be specified for importing client encrypted payloads")

Reply:
we no longer use `importToken`, nonce is what's taken its place. As to nonce/iv, noted, will fix